saleor/graphql/shop/types.py

```diff
@@ -215,7 +220,12 @@ class Shop(graphene.ObjectType):
         required=True,
     )
     display_gross_prices = graphene.Boolean(
-        description="Display prices with tax in store.", required=True
+        description="Display prices with tax in store.",
+        deprecation_reason=(
+            f"{DEPRECATED_IN_3X_FIELD} Use `Channel.taxConfiguration` to determine "
```

Comment:
Maybe? 
```suggestion
            f"{DEPRECATED_IN_3X_FIELD} Use `Channel.taxConfiguration.displayGrossPrices` to determine "
```

Reply:
`Channel.taxConfiguration.displayGrossPrices` is not the only place where you configure it; there is also a country-specific configuration (returned as a list of items) available in this object. So I think you need to check this entire object to configure it as you wish.